punkty/src/main/java/pl/fratik/punkty/LicznikPunktow.java

```diff
@@ -279,9 +279,19 @@ public void handleLvlup(LvlupEvent event) {
                 if (channelId != null && !channelId.isEmpty()) ch = shardManager.getTextChannelById(channelId);
                 if (ch == null) ch = event.getChannel();
                 if (event.getChannel().equals(ch) && !uc.isLvlupMessages()) return;
-                ch.sendMessage(tlumaczenia.get(l,
-                        "generic.lvlup", event.getMember().getUser().getName(), event.getLevel(), prefix))
-                        .queue(null, kurwa -> {});
+                if (!gc.getLvlUpMessage().isEmpty() ||gc.getLvlUpMessage() != null)  {
```

Comment:
```suggestion
                if (!gc.getLvlUpMessage().isEmpty() || gc.getLvlUpMessage() != null)  {
```

Reply:
To nie powinno byc na odwrót? Najpierw check na nulla, a później na empty.

Reply:
o wlasnie, dobrze ze zauwazyles (no i musi być "nie jest null **i** i nie jest puste")
```suggestion
                if (gc.getLvlUpMessage() != null && !gc.getLvlUpMessage().isEmpty())  {
```